src/foam/parse/FScriptParser.java

```diff
@@ -540,7 +546,7 @@ public Object value() {
       Object[] pckArr = (Object[]) values[1];
       if ( pckArr.length == 0 ) {
         try {
-          var field = this.prop_.getClassInfo().getObjClass().getDeclaredField((String) values[0]).get(null);
+          var field = this.classInfo_.getObjClass().getDeclaredField((String) values[0]).get(null);
```

Comment:
Why this change?

Reply:
FScriptParser doesn't really need a property specified, just the ClassInfo of the input object. Every time `prop_` is referred to it's only doing it to do `prop_.getClassInfo()`